# ChassisMotion.msg - 底盘运动数据
# 底盘运动信息
float64 linear_speed    # 线速度 (单位: m/s)
float64 rotate_angular  # 旋转角速度 (单位: rad/s)
float64 steering_angle  # 转向角 (单位: rad)
